presto-docs/src/main/sphinx/release/release-0.149.rst

```diff
@@ -4,10 +4,37 @@ Release 0.149
 
 General Changes
 ---------------
-* ``query.max-age`` property was renamed to ``query.min-expire-age``.
+
+* Fix planning bug that causes queries that use grouping sets over UNIONs to fail.
+* Fix issue that caused ``ARRAY_AGG`` to ignore null values.
+* Fix failure when casting row values that contain null fields.
+* Fix failure when using complex types as map keys.
+* Fix potential memory tracking leak when queries are cancelled.
+* Fix rejection of queries that do not match any queue/resource group rules.
+  Previously, a 500 error was returned to the client.
+* Fix :func:`trim` and :func:`rtrim` functions to produce more intuitive results
+  when the argument contains invalid ``UTF-8`` sequences.
+* Add a new web interface, with cluster overview, realtime stats and improved sorting
+  and filtering of queries.
+* Add support for ``FLOAT`` type.
+* Rename ``query.max-age`` to ``query.min-expire-age``.
 * ``optimizer.columnar-processing`` and ``optimizer.columnar-processing-dictionary``
   properties were merged to ``optimizer.processing-optimization`` with possible
   values ``disabled``, ``columnar`` and ``columnar_dictionary``
 * ``columnar_processing`` and ``columnar_processing_dictionary`` session
   properties were merged to ``processing_optimization`` with possible values
   ``disabled``, ``columnar`` and ``columnar_dictionary``
+* Change ``%y`` (2-digit year) in :func:`date_parse` to evaluate to a year between
+  1970 and 2069 inclusive.
+* Improve memory tracking in exchanges to avoid running out of Java heap space.
+* Improve performance of subscript operator for the ``MAP`` type.
+* Improve performance of ``JOIN`` and ``GROUP BY`` queries.
+
+Hive Changes
+------------
+
+* Clean up empty staging directories after ``INSERT INTO`` Hive table.
+* Add ``hive.ipc-ping-interval`` config for HDFS.
```

Comment:
This is a bad config name. It is specific to HDFS, so it should be named `hive.dfs.ipc-ping-interval`